shared/src/main/kotlin/org/javacs/kt/classpath/ClassPathResolver.kt

```diff
@@ -28,6 +29,15 @@ interface ClassPathResolver {
 
     val classpathWithSources: Set<ClassPathEntry> get() = classpath
 
+    /**
+     * This should return the current build file version.
+     * It usually translates to the file's lastModified time.
+     * Resolvers that don't have a build file use the default (i.e., 1).
+     * We use 1, because this will prevent any attempt to cache non cacheable resolvers
+     * (see [CachedClassPathResolver.dependenciesChanged]).
+     */
+    fun getCurrentBuildFileVersion(): Long = 1L
```

Comment:
Could we make this a property?

```suggestion
    val currentBuildFileVersion: Long
        get() = 1L
```